server/py/services/alerts/tests/unit/api/test_alerts.py

```diff
@@ -69,6 +74,48 @@ def test_store_alerts(self, db: Session, client: TestClient):
         assert len(alerts) == 1
         assert alerts[0]["name"] == alert_name
 
+    @unittest.mock.patch.object(
+        services.alerts.crud.AlertActivation, "list_alert_activations"
+    )
+    def test_list_alert_activations(
+        self, patched_list_alert_activations, db: Session, client: TestClient
+    ):
+        alert_name = "alert-name"
+        project_name = "project-name"
+
+        self._create_project(db, project_name)
+        patched_list_alert_activations.return_value = [
+            mlrun.common.schemas.AlertActivation(
+                id=1,
+                name=alert_name,
+                project=project_name,
+                severity=mlrun.common.schemas.alert.AlertSeverity.HIGH,
+                activation_time=datetime.datetime.utcnow(),
+                entity_id="1234",
+                entity_kind=mlrun.common.schemas.alert.EventEntityKind.JOB,
+                event_kind=mlrun.common.schemas.alert.EventKind.DATA_DRIFT_SUSPECTED,
+                number_of_events=1,
+                notifications=[],
+                criteria=mlrun.common.schemas.alert.AlertCriteria(count=1),
+            )
+        ]
+        # to appear in the methods which allow pagination
+        patched_list_alert_activations.__name__ = "list_alert_activations"
```

Comment:
Why not move this to a separate function that mocks the list alert activations (with default params), so we can reuse it in the future?

Reply:
@moranbental 1 - not a part of this pr, 2 - we won't need it as it's only unit test possible for alert activations (because of incompatibility of mysql and sqlite)

Reply:
@rokatyy Why can't this be part of the PR? It's about creating a new function that generates the AlertActivation with default values so we can use it later if needed.